specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -740,6 +798,32 @@
           "type": "string"
         }
       }
+    },
+    "AcsUserEngagement": {
+      "description": "The email user engagement type enum",
+      "type": "string",
+      "enum": [
+        "view",
+        "click"
+      ],
+      "x-ms-enum": {
+        "name": "UserEngagement",
+        "modelAsString": true
+      }
+    },
+    "AcsEmailStatus": {
+      "description": "The email status enum",
+      "type": "string",
+      "enum": [
+        "Failed",
+        "Delivered",
+        "FilteredSpam",
+        "Quarantined"
+      ],
+      "x-ms-enum": {
+        "name": "Status",
```

Comment:
AcsEmailStatus

Reply:
The comment from the Api View was AcsEmailDeliveryReportStatus - https://apiview.dev/Assemblies/Review/cbc0a0236ec74087b8bfa0d5e4051def/63601ccde8c34bfeafa82ac675faf684?diffRevisionId=67d693bf39c24b8f8c4632611db7d477&doc=False&diffOnly=True#Azure.Messaging.EventGrid.SystemEvents.AcsEmailDeliveryReportReceivedEventData

Reply:
I cant access this link